release/nightly_tests/nightly_tests.yaml

```
@@ -131,11 +99,27 @@
     prepare: python wait_cluster.py 4 600
     script: python shuffle/shuffle_test.py --num-partitions=200 --partition-size=500e6 --no-streaming
 
-  smoke_test:
-    cluster:
-      compute_template: shuffle/shuffle_compute_smoke.yaml  # Does not exist yet
+# Test streaming shuffle in a single node with a large partition size.
+- name: streaming_shuffle_1tb_large_partition_size
```

```suggestion
- name: streaming_shuffle_1tb_100_parts_size
```

Please fix the names to be specific here and elsewhere.